### ISSUE TITLE
bug in new version 4.4.0 sentencepiece is not available

### ISSUE THREAD

**Issue by ashaheedq** [huggingface/transformers#10749]
Hi, I am using Colab for a sentiment analysis model. 
The code suddenly stopped working after a fresh run from yesterday. 
I noticed that a new version of transformers was released which caused this issue to appear. 

when trying to import I get this error message:
`ModuleNotFoundError: No module named 'sentencepiece'`

and after installing sentencepiece using pip I get this error message:
`AttributeError: module transformers.models.ibert has no attribute IBertLayer`

**Comment by LysandreJik** [comment]
Hi, thank you for opening an issue. Could you respect the issue template? What code led to this error? What's your environment?

I can run the following without any issues:
```py
>>> from transformers import IBertModel
>>> model = IBertModel.from_pretrained("kssteven/ibert-roberta-base")
```

Thank you for your understanding.

**Comment by ashaheedq** [comment]
> Hi, thank you for opening an issue. Could you respect the issue template? What code led to this error? What's your environment?
> 
> I can run the following without any issues:
> 
> ```python
> >>> from transformers import IBertModel
> >>> model = IBertModel.from_pretrained("kssteven/ibert-roberta-base")
> ```
> 
> Thank you for your understanding.

Thanks for replying, 

I am running this Colab notebook 
https://colab.research.google.com/drive/1M0ls7EPUi1dwqIDh6HNfJ5y826XvcgGX?usp=sharing

You can reproduce by running all cells and the error will appear on the import cell.

**Comment by LysandreJik** [comment]
Thank you, I can reproduce. We'll release a patch for this in the coming days.

By the way, is there a reason you're importing everything from `transformers`, before importing specific layers?
```py
from transformers import *
from transformers import XLMRobertaConfig
from transformers import XLMRobertaModel
from transformers import AutoTokenizer, AutoModelWithLMHead
from transformers import XLMRobertaForSequenceClassification, XLMRobertaTokenizer, XLMRobertaModel
from tokenizers import Tokenizer, models, pre_tokenizers, decoders, processors
from transformers import AdamW, get_linear_schedule_with_warmup
from transformers import AutoTokenizer, AutoModel
```

**Comment by LysandreJik** [comment]
We just released version v4.4.1 with a patch for this. Thank you for letting us know!
